zk-tests/src/MockedModifier.t.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.13;
+import {Test, console2 as console} from "forge-std/Test.sol";
+
+contract MockInner {
+    // failure happens when returning >=5 items
+    function mockedMethod()
+        external
+        pure
+        returns (uint256, uint256, uint256, uint256, uint256)
+    {
+        // We fail if this function isn't mocked
```

Comment:
nit: not needed anymore?

Reply:
Think it's a nice explanation for the `assert(false)` below.

Reply:
Oh yeah sorry I misread my own explanation lol